<launch>
    <rosparam file="$(find yolo_ros)/config/config.yaml" command="load"/>
    <node name="demo_server" pkg="yolo_ros" type="demo_service_server.py" output="log"/>
    
    <node name="service_client" pkg="yolo_ros" type="service_client" output="log">
    </node>
</launch>
